hmpo-lev/email/html/template.ftl

```diff
@@ -0,0 +1,35 @@
+<#macro htmlEmailLayout>
+<html>
+  <head>
+  <style type="text/css">
+    body {
+      color: #000;
+      font-family: "HelveticaNeue", "Helvetica Neue", "Arial", "Helvetica", sans-serif;
+    }
+
+    p, li, a {
+      font-size: 19px;
+    }
+
+  </style>
+  </head>
+  <body text="#000000" bgcolor="#FFFFFF">
+    <#if realmName??>
+    <table width="100%" bgcolor="#000000" cellspacing="0" cellpadding="15">
```

Comment:
Any particular reason this is a single cell table @aarongill1 ?

Reply:
Was how it us on the parent theme...agree that it's a bit weird, so I've removed and changed to a div and added styling.

Reply:
Careful with changing the theme.

HTML for e-mails is weird.

You often have to be old-school and avoid CSS.

Reply:
I'd guess a table was used to get the padding.

Reply:
I've reverted back HTML as much as possible - research has shown using table is the preferred way for emails.